cost/low_service_usage/low_service_usage.pt

```diff
@@ -244,12 +226,12 @@ policy "policy_budget_alert" do
 | Billing Center Name | Vendor | Account Name | Service | Savings |
```

Comment:
I think there would be some value in providing the defined parameters here for billing centers and thresholds.